EnoughCoffee/src/EnoughCoffee.java

```diff
@@ -0,0 +1,51 @@
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.ArrayList;
+import java.util.Collections;
+
+public class EnoughCoffee {
+    public static void main(String[] args) throws Exception {
+        int[] array = enoughCoffee();
+        isEnough(array);
+    }
+
+    public static int[] enoughCoffee() throws IOException {
+        BufferedReader bufferedReader = new BufferedReader(new InputStreamReader(System.in));
+        ArrayList<Integer> list = new ArrayList<>();
+
+        System.out.print("Write how many ml of water the coffee machine has:\n>");
+        list.add(Integer.parseInt(bufferedReader.readLine()) / 200);
+
+        System.out.print("Write how many ml of milk the coffee machine has:\n>");
+        list.add(Integer.parseInt(bufferedReader.readLine()) / 50);
+
+        System.out.print("Write how many grams of coffee beans the coffee machine has:\n>");
+        list.add(Integer.parseInt(bufferedReader.readLine()) / 15);
+
+        System.out.print("Write how many cups of coffee you will need:\n>");
+        String count = bufferedReader.readLine();
+
+        Collections.sort(list);
```

Comment:
@AraChilingaryan why are you sorting ?

Reply:
de mer inputnery tarber en...dranq es bajanum em mer coffeein anhrajesht qanakneri vra,u sort anelov gtnum enq en minimum hat coffeen vor karox enq patrastel..karoxa stacvaci mej menq unenanq 6 bajin jur,bayc 3 bajin kat..dra hamar 3 hat kofe karox enq patrastel